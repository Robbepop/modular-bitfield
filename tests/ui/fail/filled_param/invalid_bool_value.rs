use modular_bitfield::prelude::*;

// The boolean value cannot be parsed from a string.
#[bitfield(filled = "yes")]
pub struct Base {
    a: B2,
    b: B6,
    c: u8,
    d: u32,
}

fn main() {}
