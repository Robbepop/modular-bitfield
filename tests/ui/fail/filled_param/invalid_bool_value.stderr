error: encountered invalid value argument for #[bitfield] `filled` parameter
 --> tests/ui/fail/filled_param/invalid_bool_value.rs:4:21
  |
4 | #[bitfield(filled = "yes")]
  |                     ^^^^^
